pyproject.toml

```diff
@@ -0,0 +1,79 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "MaterialX"
+version = "0.0.0.dev1"  # TODO: Change version
```

Comment:
@JeanChristopheMorinPerso Should this string match the current version of the MaterialX codebase?  If so, then we could potentially define this as a `pyproject.toml.in` file, and allow CMake to create the `pyproject.toml` file at generation time.

Reply:
Got it, and that makes sense, @JeanChristopheMorinPerso.  For now, should the string simply read "1.38.7", in preparation for potentially uploading data to PyPI?

Reply:
Yes it should. I'll update it before undrafting the PR. I also want to add a workflow to release the wheels and I prefer to keep the version like this (at least for now) to avoid mistake from my part.

Reply:
@jstone-lucasfilm The version is now fetched from `CMakeLists.txt`